src/en/readcomic/src/eu/kanade/tachiyomi/extension/en/readcomic/Readcomic.kt

```diff
@@ -0,0 +1,199 @@
+import okhttp3.Request
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class Readcomic : ParsedHttpSource() {
+
+    override val name = "ReadComic"
+
+    override val baseUrl = "https://readcomic.net"
+
+    override val lang = "en"
+
+    override val supportsLatest = true
+
+    override fun popularMangaSelector() = ".manga-box"
+
+    override fun latestUpdatesSelector() = ".home-list .hl-box .hlb-name"
+
+    override fun searchMangaSelector() = popularMangaSelector()
+
+    override fun popularMangaNextPageSelector() = ".general-nav a:last-child"
+
+    override fun latestUpdatesNextPageSelector() = popularMangaNextPageSelector()
+
+    override fun searchMangaNextPageSelector() = popularMangaNextPageSelector()
+
+    override fun popularMangaRequest(page: Int): Request {
+        return GET("$baseUrl/popular-comic/$page", headers)
+    }
+
+    override fun latestUpdatesRequest(page: Int): Request {
+        return GET("$baseUrl/comic-updates/$page", headers)
+    }
+
+    override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request =
+        GET(
+            "$baseUrl/advanced-search".toHttpUrl().newBuilder().apply {
+                addQueryParameter("key", query)
+                if (!filters.isEmpty()) {
+                    for (filter in filters) {
+                        when (filter) {
+                            is StatusFilter -> {
+                                addQueryParameter("status", arrayOf("", "CMP", "ONG")[filter.state])
+                            }
+
+                            is GenreList -> {
+                                addQueryParameter("wg", filter.included.joinToString(","))
+                                addQueryParameter("wog", filter.excluded.joinToString(","))
+                            }
+
+                            else -> {}
+                        }
+                    }
+                }
+            }.build(),
+            headers,
+        )
+
+    override fun popularMangaFromElement(element: Element): SManga =
+        SManga.create().apply {
+            setUrlWithoutDomain(element.selectFirst("a")!!.attr("abs:href"))
+            title = element.selectFirst("h3")!!.text()
+            thumbnail_url = element.selectFirst(".image")!!.selectFirst("img")!!.attr("abs:src")
+        }
+
+    override fun latestUpdatesFromElement(element: Element): SManga =
+        SManga.create().apply {
+            setUrlWithoutDomain(element.attr("abs:href"))
+            title = element.text()
+        }
+
+    override fun searchMangaFromElement(element: Element): SManga {
+        return popularMangaFromElement(element)
+    }
+
+    override fun mangaDetailsParse(document: Document): SManga = SManga.create().apply {
+        val infoElement = document.selectFirst(".manga-details tbody")
+
+        title = infoElement!!.selectFirst("td:contains(Name:)")!!.nextElementSibling()!!.text()
```

Comment:
```suggestion
        title = infoElement.selectFirst("td:contains(Name:) + td")!!.text()
```
for the rest of them too